### DISCUSSION TITLE
Allow qrcp in ufw configuration?

### DISCUSSION THREAD

**Discussion by Tup0lev** [claudiodangelis/qr-filetransfer#257] [Q&A]
Is there any way to allow qrcp through ufw firewall? 
Expected outcome: qrcp uses random port and is allowed through firewall
Attempts: make qrcp use fixed port and allow that port, but it is inconvenient to specify port every time when I use qrcp

**Comment by claudiodangelis** [comment]
Hello, thanks for reaching out!
You can easily achieve having a fixed port by setting the "port" parameter
of the JSON configuration file (please have a look at the readme).

However, a new release will be ready soon, and it will have a simpler
configuration file written in YAML, which is much better than JSON for
manual editing.

Does this help anyhow?
Cheers,
Claudio

Le jeu. 3 nov. 2022, 15:57, TupolevBlyat ***@***.***> a
écrit :
